excel-importer/src/com/axonivy/util/excel/importer/ExcelReader.java

```diff
@@ -30,36 +37,61 @@ private static List<String> getHeaderCellNames(Iterator<Row> rowIterator) {
   }
 
   private static List<Column> createEntityFields(List<String> names, Iterator<Row> rowIterator) {
-    List<Column> columns = new ArrayList<>();
+    Map<String, Column> columnMap = new LinkedHashMap<>();
```

Comment:
please move this line to line 43, because if if (!rowIterator.hasNext()) is true, then your map is created without use